cep47-logic/src/events.rs

```diff
@@ -0,0 +1,20 @@
+use std::collections::{BTreeMap, BTreeSet};
+
+use casper_types::{account::AccountHash, ContractPackageHash, Key};
+
+use crate::TokenId;
+
+pub enum CEP47Event {
```

Comment:
We need an event for BURN actions as we have burn_many, burn_one functions for NFT

Reply:
Added.